core/tests/protractor_utils/forms.js

```diff
@@ -91,10 +91,10 @@ var GraphEditor = function(graphInputContainer) {
       }
     },
     clearDefaultGraph: async function() {
-      await deleteButton.click();
+      await action.click('Delete Button', deleteButton);
       // Sample graph comes with 3 vertices.
       for (var i = 2; i >= 0; i--) {
-        await vertexElement(i).click();
+        await action.click(`Vertex Element ${i}`, vertexElement);
```

Comment:
I think you might need `vertexElement(i)` for the second argument in `action.click()`